SalesforceCommerceCloud/pwa-kit
# @W-12582733: Expose env vars endpoint for E2E smoke tests

As we roll out the new customer-defined environment variables, we must update the reference app to include this feature in our E2E smoke tests.

- [x] Created an /env-vars endpoint that returns a subset of environment variables set in Lambda

# How to Test-Drive This PR

1. npm build and start the app
1. Navigate to http://localhost:3000/ and ensure it returns a 200 no customer set variables an empty object
1. Push this app to a project that has env-vars enabled, or verify that env vars show up[ in the test site ](https://p5t-e9t.mobify-storefront-staging.com/). If using the test project, the following env vars should be returned:

```
    'customer_defined_array',
    'customer_defined_boolean',
    'customer_defined_float',
    'customer_defined_integer',
    'customer_defined_json',
    'customer_defined_null',
    'customer_defined_string',
```
